app/script/view_model/panel/ConversationParticipantsViewModel.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.viewModel = z.viewModel || {};
+window.z.viewModel.panel = z.viewModel.panel || {};
+
+z.viewModel.panel.ConversationParticipantsViewModel = class ConversationParticipantsViewModel {
+  constructor(panelViewModel, repositories) {
+    this.panelViewModel = panelViewModel;
+    this.isVisible = this.panelViewModel.conversationParticipantsVisible;
+    const conversationEntity = repositories.conversation.active_conversation;
```

Comment:
This is not a conversation entity but an oberservable containing it. Let's be more obvious in the naming.

I would go with the `activeConvesation` moniker from the repo.